.travis.yml

```diff
@@ -3,25 +3,47 @@
 # All rights reserved.
 # END COPYRIGHT BLOCK
 
+language: java
+
 services:
   - docker
 
-env:
-  - FEDORA=27
-  - FEDORA=28
+stages:
+  - validate
+  - build
+
+jobs:
+  include:
+    # Validate style of scripts
+    - stage: validate
+      env:
+        - BASE_IMAGE="stylecheck"
```

Comment:
Is it possible to use the "Build stage Matrix Expansion" feature? https://docs.travis-ci.com/user/build-stages/matrix-expansion/ and https://docs.travis-ci.com/user/build-stages#build-stages-and-build-matrix-expansion

I haven't myself implemented it in the `pki` repo, but I would love to. This feature was released recently. Since different jobs in the `build` stage have the same exact script, how about trying this out and keeping the .travis.yml short and easy maintenance?

**Note:** If you decide to implement this feature, I'm not sure whether the first stage can be named as `validate`. It defaults to `Test` as mentioned in the examples

Reply:
I'll have to take a look at that and get back to you on that. But looks interesting. :)